site2/docs/adaptors-kafka.md

```diff
@@ -128,20 +127,20 @@ Properties:
 
 | Config property                         | Supported | Notes                                                                         |
 |:----------------------------------------|:----------|:------------------------------------------------------------------------------|
-| `acks`                                  | Ignored   | Durability and quorum writes are configured at the namespace level            |
-| `auto.offset.reset`                     | Yes       | Will have a default value of `latest` if user does not give specific setting. |
+| `acks`                                  | Ignored   | Durability and quorum writes are configured at the namespace level.            |
+| `auto.offset.reset`                     | Yes       | If a user does not specify the setting, the default value is `latest`. |
 | `batch.size`                            | Ignored   |                                                                               |
-| `block.on.buffer.full`                  | Yes       | If true it will block producer, otherwise give error                          |
-| `bootstrap.servers`                     | Yes       | Needs to point to a single Pulsar service URL                                 |
+| `block.on.buffer.full`                  | Yes       | When the property is set to `true`, it blocks producer; otherwise, an error is displayed.                          |
+| `bootstrap.servers`                     | Yes       | Points to a single Pulsar service URL.                                 |
```

Comment:
The caveat `Points to a single Pulsar service URL.` is actually not true anymore, so we can remove the note now.

Reply:
@merlimat Thank you very much for your advice. 
@sijie @jiazhai could you please confirm this?

Reply:
@Jennifer88huang. right, it is as Matteo mentioned. pulsar also support servers. we could remove this note